target/linux/lantiq/xrx200/base-files/etc/board.d/02_network

```diff
@@ -43,6 +43,10 @@ lantiq_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0:lan:3" "1:lan:4" "2:lan:2" "4:lan:1" "6t@eth0"
 		;;
+	avm,fritz3390)
+		ucidef_add_switch "switch0" \
+			"0:lan:3" "1:lan:4" "2:lan:2" "4:lan:1" "5:lan:5" "6t@eth0"
```

Comment:
Do you really have five LAN ports and no wan?

Reply:
Yes and now: Physically, there are only 4 LAN parts and no WAN, but one of the ports (number 4 IIRC) is the internal connection to the ath79 target. Thus, I considered it a LAN port. Should I add a comment?

Reply:
I'm not sure whether this is the right way to do that, but again I'm lacking the technical expertise here, so I cannot give you a precise pointer.

Nevertheless, note that the number _after_ :lan: will be the port number displayed in Luci. So, if internal port no. 4 is just the internal connection, your external port numbers printed on the case should be 2-5 then, which I suspect is wrong.

Reply:
True, then it is probably port 5, the physical ports are attached to the same internal ports as for the 3370 model.

Since my current approach is to run a separate OpenWrt instance on the ath79 target, assigning it to LAN and mapping it to Port 5 by default makes sense to me.